plenum/test/view_change/test_view_change_with_instance_change_lost_due_to_restarts.py

```diff
@@ -34,7 +34,8 @@ def test_view_change_with_instance_change_lost_due_to_restarts(looper, txnPoolNo
 
     def check_ic_delivery():
         for node in txnPoolNodeSet:
-            assert node.view_changer.instanceChanges._votes_count(current_view_no + 1) == 2
+            assert current_view_no + 1 in node.view_changer.instance_changes._cache
```

Comment:
Can we use API method, like `has_view` or `has_inst_chng_from` for these checks?

Reply:
Yes, we can use ```items()``` for checking count of insurance change messages. I will change it.

Reply:
Done.